### PR TITLE
Appendable and Subtractable

### PR DESCRIPTION
**Pull request by louthy**

### CODE REVIEW

File: LanguageExt.Core/Try.cs [hunk 1]
```diff
@@ -94,6 +94,24 @@ public static class TryConfig
 /// </summary>
 public static class __TryExt
 {
+    public static Try<T> Append<T>(this Try<T> lhs, Try<T> rhs) => () =>
+    {
+        var lhsRes = lhs.Try();
+        if (lhsRes.IsFaulted) return lhsRes;
+        var rhsRes = rhs.Try();
+        if (rhsRes.IsFaulted) return lhsRes;
```
Comment (IDisposable) [inline LanguageExt.Core/Try.cs]: return rhsRes;

File: LanguageExt.Core/Try.cs [hunk 2]
```diff
@@ -94,6 +94,24 @@ public static class TryConfig
...
+        return TypeDesc.Append(lhsRes.Value, rhsRes.Value, TypeDesc<T>.Default);
+    };
+
+    public static Try<T> Subtract<T>(this Try<T> lhs, Try<T> rhs) => () =>
+    {
+        var lhsRes = lhs.Try();
+        if (lhsRes.IsFaulted) return lhsRes;
+        var rhsRes = rhs.Try();
+        if (rhsRes.IsFaulted) return lhsRes;
```
Comment (IDisposable) [inline LanguageExt.Core/Try.cs]: return rhsRes;

File: LanguageExt.Core/TryOption.cs [hunk 3]
```diff
@@ -126,6 +126,24 @@ internal TryOptionNoneUnitContext(TryOption<T> option, Action<T> someHandler, Ac
 /// </summary>
 public static class __TryOptionExt
 {
+    public static TryOption<T> Append<T>(this TryOption<T> lhs, TryOption<T> rhs) => () =>
+    {
+        var lhsRes = lhs.Try();
+        if (lhsRes.IsFaulted || lhsRes.Value.IsNone) return lhsRes;
+        var rhsRes = rhs.Try();
+        if (rhsRes.IsFaulted || rhsRes.Value.IsNone) return lhsRes;
```
Comment (IDisposable) [inline LanguageExt.Core/TryOption.cs]: return rhsRes;

File: LanguageExt.Core/TryOption.cs [hunk 4]
```diff
@@ -126,6 +126,24 @@ internal TryOptionNoneUnitContext(TryOption<T> option, Action<T> someHandler, Ac
...
+        return TypeDesc.Append(lhsRes.Value, rhsRes.Value, TypeDesc<T>.Default);
+    };
+
+    public static TryOption<T> Subtract<T>(this TryOption<T> lhs, TryOption<T> rhs) => () =>
+    {
+        var lhsRes = lhs.Try();
+        if (lhsRes.IsFaulted || lhsRes.Value.IsNone) return lhsRes;
+        var rhsRes = rhs.Try();
+        if (rhsRes.IsFaulted || rhsRes.Value.IsNone) return lhsRes;
```
Comment (IDisposable) [inline LanguageExt.Core/TryOption.cs]: return rhsRes;

**Comment by louthy** [comment]
Fixed

**Pull request merged**